codegen/src/test/kotlin/software/amazon/smithy/rust/codegen/smithy/protocols/parse/XmlBindingTraitParserGeneratorTest.kt

```diff
@@ -89,7 +89,7 @@ internal class XmlBindingTraitParserGeneratorTest {
     """.asSmithyModel()
 
     @Test
-    fun `generates valid parsers`() {
+    fun `generates valid parsers in client`() {
```

Comment:
Why this change?

Reply:
In revision 0, I had two completely independent tests. Sorry this remained from that revision